cypress/integration/seededFlows/mobileFlows/namespacedForemMobile.spec.js

```diff
@@ -0,0 +1,112 @@
+// This E2E test focuses on ensuring the mobile bridge integration
+describe('Namespaced ForemMobile functions', () => {
+  function waitForBaseDataLoaded() {
+    cy.get('body').should('have.attr', 'data-loaded', 'true');
+  }
+
+  // Make the runtime act as ForemWebView context
+  const runtimeStub = {
+    onBeforeLoad: (win) => {
+      Object.defineProperty(win.navigator, 'userAgent', {
+        value:
+          'Mozilla/5.0 (iPhone; CPU iPhone OS 14_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 ForemWebView/1.0',
+      });
+      Object.defineProperty(win.navigator, 'platform', { value: 'iPhone' });
+    },
+  };
+
+  describe('within ForemWebView context', () => {
```

Comment:
Curious if you figured out why some of these tests require a user to be logged in, e.g. `injectMessage`.